lib/sessionParticipants/index.js

```diff
@@ -9,13 +9,16 @@ function SessionParticipants (sessions) {
  * represents the issuer of the previous mentions request/response.
  * 
  * @issuer {string}   The string as it was received in the SAML request/response
+ * @sessionIndex {string}   The string as it was received in the SAML request/response. Only available in LogoutRequests
  * @cb     {function} The callback that will be called with '(err, sessionParticipant)'
  */
-SessionParticipants.prototype.get = function(issuer, cb) {
+SessionParticipants.prototype.get = function(issuer, sessionIndex, nameId, cb) {
+  // SessinIndex should be mandatory, but not issuer
+  // Let's keep using issuer only if available
   var s = this._participants.find(function (session) {
-    return session.serviceProviderId === issuer;
+    return session.serviceProviderId === issuer  && session.sessionIndex === sessionIndex  && session.nameId === nameId;
```

Comment:
They shouldn't, as nameid is mandatory. If the three of them are undefined, it won't find any session, throwing an error